community/dags/dag_stock_comments.py

```
@@ -0,0 +1,90 @@
+        crawler.close()
+        logger.info(f"✅ {stock_symbol} 크롤링 완료: {len(comments) if comments else 0} 댓글")
+        return len(comments) if comments else 0
+    except Exception as e:
+        logger.error(f"❌ {stock_symbol} 크롤링 실패: {e}")
+        raise
+
+def validate_data(**context):
+    logger = setup_logger('validate_data', logs_base_dir)
+    try:
+        with psycopg2.connect(**db_config) as conn:
+            with conn.cursor() as cur:
```

Ruff (0.12.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)